config/application.rb

```diff
@@ -9,7 +9,8 @@
 
 module Epigaea
   class Application < Rails::Application
-    config.active_job.queue_adapter = :sidekiq
+    # config.active_job.queue_adapter = :sidekiq
+    config.active_job.queue_adapter = :inline
```

Comment:
Can you say something about this? I'm thinking this looks like an artifact of your development env?

Reply:
You are correct. Whoops. :(